source/authors.rst

```diff
@@ -41,6 +41,11 @@ The following people have contributed to the Tango documentation over the years:
   Piotr Goryl, Olga Merkulova, Lukasz Zytniak, Lukasz Dudek, Matteo di Carlo,
   Matteo Canzari, Igor Khokhrakiov, Reynald Bourtembourg, Jean-Michel Chaize, 
   Stuart James and Andy Götz
+* **Second Write-the-Doc Team** - the following people participated in the second Write-the-Doc camp:
```

Comment:
@andygotz, at first I proposed the list according to all who sing-up for the DocCamp. However, should we limit the list to ones who really participate (by either writing or commenting or discussing) in the documentation?
If yes, it should be:
Reynald Bourtembourg, Thomas Braun, 
  Sébastien Gara, Philippe Gauron, Andrew Götz, Piotr Goryl, 
  Anton Joubert, Krystian Kędroń, Igor Khokhrakiov, Grzegorz Kowalski, Olga Merkulova,
  Guillaume Mugerin, Lorenzo Pivetta, Sergi Rubio

Reply:
@piogor I agree we should limit it to the people who really contributed. Thanks!

Reply:
Thanks! Done.